src/pages/ProductDetail/ProductDescription/ProductDescription.js

```diff
@@ -1,43 +1,53 @@
-import React, { Component } from 'react';
-import './ProductDescription.scss';
+import React, { Component } from "react";
+import "./ProductDescription.scss";
 
 class ProductDescription extends Component {
-    constructor() {
-        super();
-        this.state = {
-            DescriptionBtnState: 2,
-        }
-    }
-
-    openDescription = (e) => {
-        const { isDescriptionOpen } = this.state;
-        this.setState({ isDescriptionOpen: !isDescriptionOpen });
+  constructor() {
+    super();
+    this.state = {
+      DescriptionBtnState: 2,
     };
+  }
+
+  openDescription = (e) => {
+    const { isDescriptionOpen } = this.state;
+    this.setState({ isDescriptionOpen: !isDescriptionOpen });
+  };
 
-    hadleClick = async (e) => {
-        const { DescriptionBtnState } = this.state;
-        const { name } = e.target;
-        this.setState(DescriptionBtnState===name?{ DescriptionBtnState:2 }:{ DescriptionBtnState: name });
-    }
+  hadleClick = async (e) => {
+    const { DescriptionBtnState } = this.state;
+    const { name } = e.target;
+    this.setState(
+      DescriptionBtnState === name ? { DescriptionBtnState: 2 } : { DescriptionBtnState: name },
+    );
```

Comment:
```suggestion
    this.setState({ DescriptionBtnState: DescriptionBtnState === name ? 2 : name });
```
삼항연산자를 적절한 위치에 구사해주시면 더 깔끔한 코드를 짤 수 있습니다